frontend/package.json

```
@@ -61,6 +63,7 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "nuxt prepare",
     "preview": "nuxt preview",
+    "test:e2e": "npx playwright test --config=tests/playwright.config.ts",
```

this will come in handy I guess
```suggestion
    "test:e2e": "npx playwright test --config=tests/playwright.config.ts",
     "test:e2e:ui": "npm run test:e2e -- --ui",
```